src/main/kotlin/org/opensearch/commons/alerting/model/DataSources.kt

```diff
@@ -31,6 +31,12 @@ data class DataSources(
     /** Configures a custom index pattern for alertHistoryIndex alias.*/
     val alertsHistoryIndexPattern: String? = "<.opendistro-alerting-alert-history-{now/d}-1>", // AlertIndices.ALERT_HISTORY_INDEX_PATTERN
 
+    /** Configures a custom index alias to store comments associated with alerts.*/
+    val commentsIndex: String = ".opensearch-alerting-comments-history-write", // AlertIndices.COMMENTS_HISTORY_WRITE_INDEX
```

Comment:
this should be optional?? 
existing usage of dataSources like SEcurity analytics will not be setting this field.